src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -157,7 +158,8 @@ private JinjavaConfig(Builder builder) {
 
   private ObjectMapper setupObjectMapper(@Nullable ObjectMapper objectMapper) {
     if (objectMapper == null) {
-      objectMapper = new ObjectMapper();
+      objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
```

Comment:
What if ObjectMapper is not null? Shouldn't you register JavaTimeModule in that case?

Reply:
Yes, that's a good point. This will be addressed by the change Julia suggested.